agent/rpm/pbench-agent.spec.j2

```diff
@@ -23,6 +29,14 @@ Requires:  python3-cffi, python3-click, python3-requests
 Requires:  python3-docutils, python3-psutil
 %endif
 
+%if 0%{?rhel} == 9
+Requires:  python3-pip
+# RPMs for modules in requirements.txt
+Requires:  python3-cffi, python3-requests
+# RPMs for module dependencies
```

Comment:
What does "module dependencies" mean? Are we explicitly requiring indirect requirements? If so, it might be nice (if possible) to mention which modules place these requirements to make it easier in the future to track them.

Reply:
I'm actually not sure and I did not stop to investigate who uses the `docutils` and `psutil` modules. This was cut-and-pasted from RHEL8. I'll open an issue.

Reply:
Opened #2583.

Reply:
[Blame](https://github.com/distributed-system-analysis/pbench/blame/13c20cbe3dfefed66fd8ddb32498c204233c3461/agent/rpm/pbench-agent.spec.j2) suggests that some guy named @portante  [put that code comment](https://github.com/distributed-system-analysis/pbench/commit/7f614fe041886a0fea352710cacd82d272ad2d75) in there as a replacement for the phrase "derived dependencies".